apps/communications/contacts/js/contacts.js

```diff
@@ -304,6 +304,11 @@ var Contacts = (function() {
       if (activityName === 'pick' || activityName === 'update') {
         alternativeTitle = 'selectContact';
       }
+      if (ActivityHandler.currentActivityIs(['pick']) &&
```

Comment:
better if we define a function that is able to map an activityName and activityType to a CSS class to be added to the container. That way the code will be more generic and less polluted